pennylane/labs/tests/resource_estimation/test_resource_operator.py

```diff
@@ -37,7 +37,7 @@
     _make_hashable,
     resource_rep,
 )
-from pennylane.queuing import AnnotatedQueue
+from pennylane.queuing import AnnotatedQueue, QueuingManager
```

Comment:
CF saying it is not needed? 
```suggestion
from pennylane.queuing import AnnotatedQueue
```

Reply:
Ya! nice catch 👍🏼